README.md

```diff
@@ -321,66 +321,100 @@ Fidel.setOptions({
 });
 ```
 
-### programName
+### deleteInstructions
 
-Set your program name as it will appear in the consent text. Note that this parameter is used when you set United States or Canada as the default country or don't set a default country (meaning that the user is free to select United States or Canada as their country). Please set it to a maximum of 60 characters.
+Write your custom opt-out instructions for your users. They will be displayed in the consent checkbox text as well.
 
 ```javascript
 Fidel.setOptions({
-  programName: 'Your Program Name'
+  deleteInstructions: 'Your custom card delete instructions!'
 });
 ```
 
-### deleteInstructions
+### privacyUrl
 
-Write your custom opt-out instructions for your users. They will be displayed in the consent checkbox text as well.
+This is the privacy policy URL that you can set for the consent checkbox text.
 
 ```javascript
 Fidel.setOptions({
-  deleteInstructions: 'Your custom card delete instructions!'
+  privacyUrl: 'https://fidel.uk',
 });
 ```
 
-### privacyUrl
+### programName (USA&Canada only)
 
-This is the privacy policy URL that you can set for the consent checkbox text.
+Set your program name as it will appear in the consent text. Note that **this parameter is optional** and used when you set United States or Canada as the default country or don't set a default country (meaning that the user is free to select United States or Canada as their country). Please set it to a maximum of 60 characters.
 
 ```javascript
 Fidel.setOptions({
-  privacyUrl: 'https://fidel.uk',
+  programName: 'Your Program Name'
 });
 ```
 
-### termsConditionsUrl
+### termsConditionsUrl (USA&Canada only)
```

Comment:
Maybe in parenthesis: "Applied to the consent text, only for USA and Canada." Please write the same for the "programName", above.